docs/src/prototypes/CopyToClipboard/NotificationProvider.tsx

```diff
@@ -28,9 +35,21 @@ export const NotificationProvider: React.FC = props => {
 
   return (
     <>
-      <Portal open={!!notification}>
-        <Notification>{notification}</Notification>
-      </Portal>
+      {!!targetRef ? (
+        !!notification && (
+          <Tooltip
+            content={notification}
+            pointing={true}
```

Comment:
It doesn't respect `pointing` value from the parent component

Reply:
according to design, this should never be pointing, so I will always set it to false